test/e2e/test_job_submissions.py

```diff
@@ -32,6 +32,58 @@
 @pytest.mark.usefixtures("session_worker")
 @pytest.mark.parametrize("operating_system", [os.environ["OPERATING_SYSTEM"]], indirect=True)
 class TestJobSubmission:
+    def test_success(
+        self,
+        deadline_resources,
+        deadline_client: DeadlineClient,
+    ) -> None:
+        # WHEN
+        job = Job.submit(
+            client=deadline_client,
+            farm=deadline_resources.farm,
+            queue=deadline_resources.queue_a,
+            priority=98,
+            template={
+                "specificationVersion": "jobtemplate-2023-09",
+                "name": "Test Success Sleep Job",
+                "steps": [
+                    {
+                        "hostRequirements": {
+                            "attributes": [
+                                {
+                                    "name": "attr.worker.os.family",
+                                    "allOf": [os.environ["OPERATING_SYSTEM"]],
+                                }
+                            ]
+                        },
+                        "name": "Step0",
+                        "script": {
+                            "actions": {
+                                "onRun": {
+                                    "command": (
+                                        "/bin/sleep"
+                                        if os.environ["OPERATING_SYSTEM"] == "linux"
+                                        else "powershell"
+                                    ),
+                                    "args": (
+                                        ["40"]
```

Comment:
Do we need to sleep 40 seconds if we just want to test a successful job? This would make the tests run longer than necessary.